src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/ChunkedEncodingReadStream.cs

```diff
@@ -279,6 +289,7 @@ private ReadOnlyMemory<byte> ReadChunkFromConnectionBuffer(int maxBytesToRead, C
 
                             while (true)
                             {
+                                // Folded header was deprecated in RFC 7230 3.2.4. Thus we don't support it.
```

Comment:
We do support folded headers, e.g.
https://github.com/dotnet/corefx/blob/caba0ef9ea05f66aaae814e89848acc98c23f327/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs#L928-L949
Is this saying we don't support trailing folded headers?

Reply:
In the old [RFC 2616](https://tools.ietf.org/html/rfc2616#section-3.6.1), trailer is of `entity-header` type - which is strictly disallowed in the new RFC 7230.

In the new RFC 7230, trailer now is of `header-field` type - also in the same RFC, folded header is obsoleted in `header-field`. So we should not support folded trailing header.

Reply:
I don't see how folded headers are different between regular headers and trailing headers in RFC 7230. It's true that folded headers are obsoleted, but since we support them for regular headers, why wouldn't we support them for trailing headers too?

Reply:
Turns out `entity-header` is a subset of `header-field`. Will add folded header support in next commit.